.gitmodules

```diff
@@ -28,8 +28,8 @@
   branch = dev/emc
 [submodule "CICE"]
   path = CICE-interface/CICE
-  url = https://github.com/NOAA-EMC/CICE
-  branch = emc/develop
+  url = https://github.com/CICE-Consortium/CICE
```

Comment:
Does this mean we will use the CICE consortium main branch directly in UFS WM? That means CICE consortium code managers will get involved in the EMC cice code updates. Can we still use EMC's cice fork with a new develop branch? e.g.

url = https://github.com/NOAA-EMC/CICE
  branch = develop

Reply:
I only used Consortium branch for testing b4b. Yes, we will use a EMC/CICE develop branch in UFS

Reply:
I just made https://github.com/NOAA-EMC/CICE/tree/develop from CICE-Consortium. I didn't realize I had permissions